### ISSUE TITLE
Poor legibility of Title of label window in search results of dark theme

### ISSUE THREAD

**Issue by earendil1** [nextcloud/server#35278]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**
When one has switched the dark theme on (accessibility), the label window in search results is poorly legible.

**Describe the solution you'd like**
The words "Please select tag(s) to add to the selection" should be in a darker font.

**Describe alternatives you've considered**
Not really trying to read the text as it is hard to read.

**Additional context**
Screenshot
![Schermata 2022-11-19 alle 12 35 17](https://user-images.githubusercontent.com/76689803/202848723-24194b9f-fcdc-4a7f-84b3-246fcb003fcb.png)

NC 24.0.5

**Comment by svtcpxdev** [comment]
I've never been able to use the built in dark theme for several such issues. The ppp "Breeze Dark" brings you a much nicer implementation of a dark mode in NC, you might want to checkt that out. Thank you for not being a light mode goblin!

**Comment by earendil1** [comment]
Hi @svtcpxdev!
Thank you for the suggestion. It loaded it all right. For this issue, I think there is no improvement, though:
![Schermata 2022-11-21 alle 15 00 19](https://user-images.githubusercontent.com/76689803/203073719-6630519a-4fe7-46dc-a757-90e6b8d5fd2c.png)

**Comment by earendil1** [comment]
I think so, here is a screenshot: ![Schermata 2022-11-21 alle 15 39 52](https://user-images.githubusercontent.com/76689803/203082529-2cb6c158-7c37-4e49-b5b4-8d387c8da029.png)

**Comment by svtcpxdev** [comment]
Looks about right, yes. In that case, only some custom CSS could help you with your issue. Do you know how to add it?

**Comment by earendil1** [comment]
I actually do not, I am very inexperienced. I would not know what or where to add it.
Would it affect this particular part of Nextcloud? Only my instance? Or is it something like a contribution to the code for the benefit of all? This was ultimately the meaning of my initial comment.

**Comment by susnux** [comment]
This is fixed with NC25 (at least with 25.0.3)

**Comment by joshtrichards** [comment]
Hmm, I can reproduce this still in more recent NC25 as well as NC26 and NC27. Can't confirm either way in `master` at the moment since tags aren't visible for me for some reason...
